### DISCUSSION TITLE
https://docs.requarks.io/install/docker

### DISCUSSION THREAD

**Discussion by jwilleke** [requarks/wiki#3641] [Error / Bug Report]
**Describe the bug**
User Docker compase on MacOS <https://docs.requarks.io/install/docker>

**To Reproduce**
1. Create docker-compose.yaml use file as base for installation
2. sudo docker-compose run wikitest .
3. Screen shows
```Creating docker-wikijs_db_1 ... done
Pulling wikitest (requarks/wiki:2)...
...
c9b1b535fdd9: Already exists
32eb17722c57: Pull complete
c6a4a31e8940: Pull complete
3f01d5292e29: Pull complete
6af8937c9b75: Pull complete
4f4fb700ef54: Pull complete
98abbb1524d0: Pull complete
38ada7e448dc: Pull complete
673be8a35263: Pull complete
88360fb8e475: Pull complete
f15a3ece6568: Pull complete
e7c28b1aaa84: Pull complete
b76a16c3b4c0: Pull complete
d8210702a67e: Pull complete
Digest: sha256:3777e176aa285fcf4c7b6064bf8e64919de3d5ff0ea13d3e009ec8145f5b2056
Status: Downloaded newer image for requarks/wiki:2
/usr/local/bin/docker-entrypoint.sh: exec: line 8: .: Permission denied
```
It was suggested that the docker-entrypoint.sh files is NOT executable.

**Expected behavior**
The wiki.js to run

**Screenshots**

**Host Info (please complete the following information):**
 - OS: 
ProductName:	Mac OS X
ProductVersion:	10.15.3
BuildVersion:	19D76
 - Wiki.js version: requarks/wiki:2
 - Database engine: postgres:11-alpine

**Additional context**

**Comment by Smankusors** [comment]
I cannot reproduce this on Ubuntu, I guess it's only on MacOS? What is the output when you run this command?

```
docker-compose exec wiki ls -lah /usr/local/bin
```

**Comment by jwilleke** [comment]
What I see on MacOS:
```
sudo docker-compose run wikitest ls -lah /usr/local/bin
Starting docker-wikijs_db_1 ... done
total 44M
drwxr-sr-x    1 root     root        4.0K Jan 18 04:22 .
drwxr-xr-x    1 root     root        4.0K Jan 18 04:22 ..
-rwxrwxr-x    1 root     root         116 Jan 18 04:22 docker-entrypoint.sh
-rwxr-xr-x    1 root     root       44.1M Jan  7 18:06 node
lrwxrwxrwx    1 root     root          19 Jan 18 04:22 nodejs -> /usr/local/bin/node
lrwxrwxrwx    1 root     root          38 Jan 18 04:22 npm -> ../lib/node_modules/npm/bin/npm-cli.js
lrwxrwxrwx    1 root     root          38 Jan 18 04:22 npx -> ../lib/node_modules/npm/bin/npx-cli.js
lrwxrwxrwx    1 root     root          26 Jan 18 04:22 yarn -> /opt/yarn-v1.21.1/bin/yarn
lrwxrwxrwx    1 root     root          29 Jan 18 04:22 yarnpkg -> /opt/yarn-v1.21.1/bin/yarnpkg
```

**Comment by jwilleke** [comment]
When I run:
```
sudo docker-compose exec wikitest ls -lah /usr/local/bin
ERROR: No container found for wikitest_1
```

**Comment by jwilleke** [comment]
Using:
<https://github.com/jwilleke/docker-wikijs/blob/master/docker-compose.yaml>

**Comment by Smankusors** [comment]
Oh yeah it should be `run` command because the container itself failed to start...  but the execute bit permission already there. Could be something wrong with your Docker / MacOS config? What is your Docker version?

**Comment by jwilleke** [comment]
docker -v
Docker version 19.03.5, build 633a0ea

**Comment by Smankusors** [comment]
btw with Wiki.js 2.2.51, is this still happen?

**Comment by jwilleke** [comment]
Appears it does.

```sudo docker-compose run wikitest .
Starting docker-wikijs_db_1 ... done
Pulling wikitest (requarks/wiki:2.2.51)...
2.2.51: Pulling from requarks/wiki
c9b1b535fdd9: Already exists
6ad7c27326e7: Pull complete
968d8b4948da: Pull complete
a1ec837a6331: Pull complete
c12f659e9fb4: Pull complete
4f4fb700ef54: Pull complete
72bb64ccfa2a: Pull complete
de12dad5095a: Pull complete
a8a7e7fd0abc: Pull complete
46873461ab98: Pull complete
0c4edf746459: Pull complete
94dee7f38f8a: Pull complete
5a553c06aecd: Pull complete
Digest: sha256:3a3575554b90b190ad7683fcf77e4f83dee7e218fe2dec70888ca026c0371739
Status: Downloaded newer image for requarks/wiki:2.2.51
/usr/local/bin/docker-entrypoint.sh: exec: line 8: .: Permission denied
```
Mac OS X ProductVersion:	10.15.4 BuildVersion:	19E287

**Comment by sbonaime** [comment]
@jwilleke and if you put sudo before ?

**Comment by Smankusors** [comment]
> Appears it does.
> 
> ```
> Starting docker-wikijs_db_1 ... done
> Pulling wikitest (requarks/wiki:2.2.51)...
> 2.2.51: Pulling from requarks/wiki
> c9b1b535fdd9: Already exists
> 6ad7c27326e7: Pull complete
> 968d8b4948da: Pull complete
> a1ec837a6331: Pull complete
> c12f659e9fb4: Pull complete
> 4f4fb700ef54: Pull complete
> 72bb64ccfa2a: Pull complete
> de12dad5095a: Pull complete
> a8a7e7fd0abc: Pull complete
> 46873461ab98: Pull complete
> 0c4edf746459: Pull complete
> 94dee7f38f8a: Pull complete
> 5a553c06aecd: Pull complete
> Digest: sha256:3a3575554b90b190ad7683fcf77e4f83dee7e218fe2dec70888ca026c0371739
> Status: Downloaded newer image for requarks/wiki:2.2.51
> /usr/local/bin/docker-entrypoint.sh: exec: line 8: .: Permission denied
> ```
> 
> Mac OS X ProductVersion: 10.15.4 BuildVersion: 19E287

did this happen only on Wiki.js? How about other container? mysql? or node.js container?
